endpoints.json

```diff
@@ -140,8 +140,23 @@
                 }
               ]
             }
-          }
-        } 
-        
-      
-      
\ No newline at end of file
+          },
+          "GET /api/articles?topic=cats": {
+            "description": "Serves an array of articles filtered by the query of topic",
+            "queries": ["author","title","article_id","topic","created_at","votes","article_img_url","comment_count"],
+            "exampleResponse": {
+                "articles": [
+                    {
+                        "article_id": 5,
+                        "title": "UNCOVERED: catspiracy to bring down democracy",
+                        "topic": "cats",
+                        "author": "rogersop",
+                        "created_at": "2020-08-03T13:14:00.000Z",
+                        "votes": 0,
+                        "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                        "comment_count": "2"
+                    }
+                ]
+            }
+        }
```

Comment:
This should not be added as a new entry to endpoints.json. You should already have a get /api/articles endpoint you need to add a query of topic to that array and that is all you will need to do for this endpoint.